crates/router/src/connector/aci/transformers.rs

```diff
@@ -141,7 +157,25 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for AciPaymentsRequest {
                 card_cvv: ccard.card_cvc,
             })),
             api::PaymentMethodData::PayLater(_) => PaymentDetails::Klarna,
-            api::PaymentMethodData::Wallet(_) => PaymentDetails::Wallet,
+            api::PaymentMethodData::Wallet(ref wallet_data) => match wallet_data {
+                api_models::payments::WalletData::MbWay(data) => {
+                    PaymentDetails::Wallet(Box::new(WalletPMData {
+                        payment_brand: PaymentBrand::Mbway,
+                        account_id: Some(data.telephone_number.clone()),
+                        shopper_result_url: item.request.router_return_url.clone(),
+                    }))
+                }
+                api_models::payments::WalletData::AliPay { .. } => {
+                    PaymentDetails::Wallet(Box::new(WalletPMData {
+                        payment_brand: PaymentBrand::Alipay,
```

Comment:
Please use consistent naming across the codebase: either `Alipay` or `AliPay` but not both.

Reply:
Naming is being done in consideration to how field are being accepted by ACI(SCREAMING_SNAKE_CASE), ALIPAY was the acceptable parameter, so I had to provide it as Alipay for the casing. Would it be ok to make change in payment.rs for the consistency?

Reply:
If another connector expects it as `AliPay` or `ali_pay`, we'd end up renaming in multiple places again if we want to ensure consistency. I'd like to hear what @ArjunKarthik or @jagan-jaya feel about this. If either of them are okay with how things are now, we can proceed with this.

Reply:
Had a convo with Jagan, he asked me to use "AliPay" by renaming it with serde. I will make the changes👍